public/helpers/update-users.js

```diff
@@ -0,0 +1,34 @@
+const updateUsers = (sender, users) => {
+  users.forEach((user) => {
+    if(user.permissions.length == 2){
+      if(mentorExist()){
+        addUser('moderator', user);
+      }
+      else{
+        addUser('mentor', user);
```

Comment:
I think this would need to be addUser('moderator', user) if the mentor doesn't exist